integration-tests/src/test/resources/sitconfig/java/SitConfigTests.java

```diff
@@ -0,0 +1,554 @@
+    int got = serverMBean.getConnectTimeout();
+    assert expectedValue == got
+        : "Didn't get the expected value " + expectedValue + " for ConnectTimeout";
+  }
+
+  protected void verifyRestartMax(int expectedValue) {
+    ServerMBean serverMBean = getServerMBean();
+    int got = serverMBean.getRestartMax();
+    assert expectedValue == got
+        : "Didn't get the expected value " + expectedValue + " for RestartMax";
+  }
+
+  protected void verifyMaxMessageSize(int expectedValue) {
+    ServerMBean serverMBean = getServerMBean();
+    int got = serverMBean.getMaxMessageSize();
+    assert expectedValue == got
+        : "Didn't get the expected value " + expectedValue + " for MaxMessageSize";
+  }
+
+  protected void verifyT3ChannelPublicAddress(String expectedValue) {
+    boolean got = false;
+    ServerMBean serverMBean = getServerMBean();
+    NetworkAccessPointMBean[] networkAccessPoints = serverMBean.getNetworkAccessPoints();
+    for (NetworkAccessPointMBean networkAccessPoint : networkAccessPoints) {
+      if (networkAccessPoint.getName().equals("T3Channel")) {
+        assert expectedValue.equals(networkAccessPoint.getPublicAddress())
+            : "Didn't get the expected value " + expectedValue + " for T3Channel public address";
+      }
+    }
+  }
+
+  protected void verifyT3ChannelPublicPort(int expectedValue) {
+    boolean got = false;
+    ServerMBean serverMBean = getServerMBean();
+    NetworkAccessPointMBean[] networkAccessPoints = serverMBean.getNetworkAccessPoints();
+    for (NetworkAccessPointMBean networkAccessPoint : networkAccessPoints) {
+      if (networkAccessPoint.getName().equals("T3Channel")) {
+        assert expectedValue == networkAccessPoint.getPublicPort()
+            : "Didn't get the expected value " + expectedValue + " for T3Channel public address";
+      }
+    }
+  }
+
+  protected void verifyT3ChannelMaxMessageSize(int expectedValue) {
+    boolean got = false;
+    ServerMBean serverMBean = getServerMBean();
+    NetworkAccessPointMBean[] networkAccessPoints = serverMBean.getNetworkAccessPoints();
+    for (NetworkAccessPointMBean networkAccessPoint : networkAccessPoints) {
+      if (networkAccessPoint.getName().equals("T3Channel")) {
+        assert expectedValue == networkAccessPoint.getMaxMessageSize()
+            : "Didn't get the expected value " + expectedValue + " for MaxMessageSize";
+      }
+    }
+  }
+
+  private ServerMBean getServerMBean() {
+    ServerMBean serverMBean = runtimeServiceMBean.getServerConfiguration();
+    println("ServerMBean: " + serverMBean);
+
+    return serverMBean;
+  }
+
+  public void testSystemResourcesJDBCAttributeChange(String jdbcResourceName) {
+    int initialCapacity = 2;
+    int maxCapacity = 12;
+    boolean testConnectionsonReserve = true;
+    int harvestMaxCount = 7;
+    int inactiveConnectionTimeoutSeconds = 120;
+    String dsUrl = "jdbc:oracle:thin:@//slc11smq.us.oracle.com:1583/w18ys12c.us.oracle.com";
```

Comment:
Another internal URL

Reply:
will work on setting up the mysql db and replace internal db references with mysql db

Reply:
SECURITY VIOLATION
do not check oracle internal server names into public repositories

Reply:
do you need a database to run this test - that is a problem - we need to discuss that

Reply:
replaced oracle with mysql db and removed all internal server references